lib/graph.py

```diff
@@ -46,7 +51,11 @@ def __init__(self, fyle, truncate=30, **style):
         self.style = style
 
         if len(path.name) + len(ext) > truncate:
-            self.style["label"] = path.name[:(truncate - 1 - len(ext))] + "…" + ext
+            self.style["label"] = wrap_command(
```

Comment:
Please make this consistent with the surrounding style, this codebase doesn't have newlines after each parameter. Suggest:
```python
path_name = wrap_command(truncate - 1 - len(ext), '\n', path.name)
self.style["label"] = f"{path_name}{ext}"
```

Reply:
The change to split the filenames over multiple lines is different to having a description + command in the command box, please make it a separate commit